packages/matchbox/src/components/Text/Text.js

```diff
@@ -1,28 +1,36 @@
+import React from 'react';
 import styled from 'styled-components';
+import { Box } from '../Box';
 import { color, space, typography, compose } from 'styled-system';
-import { truncate, crop } from './styles';
+import { truncate, looksLike } from './styles';
 import PropTypes from 'prop-types';
 
-const system = compose(
-  color,
-  space,
-  typography
-);
+const system = compose(color, space, typography);
 
-const Text = styled('p')`
+const StyledText = styled(Box)`
   ${system}
   ${truncate}
-  ${crop}
+  ${looksLike}
 `;
 
+const Text = function(props) {
+  const { as, looksLike, children, ...rest } = props;
+
+  return (
+    <StyledText as={as} looksLike={looksLike} {...rest}>
```

Comment:
Yeah think this is whats causing the jest error, could just rename it before passing it through and maybe add a comment so we remember why we did it

`looks={looksLike}`